src/BlockTypes/Checkout.php

```diff
@@ -82,8 +82,6 @@ public function render( $attributes = array(), $content = '' ) {
 			$data_registry->add( 'shippingMethodsExist', $methods_exist );
 		}
 
-		\Automattic\WooCommerce\Blocks\Assets::register_block_script( $this->block_name . '-frontend', $this->block_name . '-block-frontend' );
```

Comment:
Why is this redundant? where are we also including this?

Reply:
See next few lines in the file, when I did a rebase I got two copies of the same code. (Always look past the raw diff when reviewing!)